README.md

```diff
@@ -85,21 +83,14 @@ For remote servers you'll need to have a base Ubuntu 14.04 server already create
 
 ### WordPress Sites
 
-Since Trellis is all about automatically creating servers for your WordPress sites, you need to configure your sites before anything else.
-
-This configuration is done in the environment files inside the `group_vars` directory. The `group_vars` files are in [YAML](http://en.wikipedia.org/wiki/YAML) format.
-
-To configure the sites you want on your Vagrant development VM, you'd edit `group_vars/development` for example. For staging, `group_vars/staging`. And likewise for production: `group_vars/production`.
+Before using Trellis, you must configure your WordPress sites. The `group_vars` directory contains one configuration file per environment (`development`, `staging`, and `production`; in [YAML](http://en.wikipedia.org/wiki/YAML) format). For example, to configure the sites on your Vagrant development VM, edit `group_vars/development`.
```

Comment:
>For example, to configure the sites on your Vagrant development VM, edit `group_vars/development`.

For example needs a colon instead of comma

For example: configure the sites on your Vagrant development VM by editing `group_vars/development`.